src/app/+my-dspace-page/my-dspace-new-submission/my-dspace-new-submission.component.spec.ts

```diff
@@ -22,9 +22,11 @@ import { getMockScrollToService } from '../../shared/mocks/scroll-to-service.moc
 import { UploaderService } from '../../shared/uploader/uploader.service';
 import { By } from '@angular/platform-browser';
 import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
+import { HostWindowService } from '../../shared/host-window.service';
+import { HostWindowServiceStub } from '../../shared/testing/host-window-service.stub';
 import { UploaderComponent } from 'src/app/shared/uploader/uploader.component';
 
-describe('MyDSpaceNewSubmissionComponent test', () => {
+fdescribe('MyDSpaceNewSubmissionComponent test', () => {
```

Comment:
@sourcedump : The large code coverage drop is caused by this single line.  The `fdescribe` here adds a focus to this specific test & ensures that **only focused tests run**.  See https://jasmine.github.io/2.2/focused_specs.html

Reply:
fixed